src/ui/io.c

```diff
@@ -7,50 +12,52 @@
 
 unsigned char G_io_seproxyhal_spi_buffer[ IO_SEPROXYHAL_BUFFER_SIZE_B ];
 
+#ifdef HAVE_BAGL
 void io_seproxyhal_display(const bagl_element_t *element) {
     io_seproxyhal_display_default((bagl_element_t *)element);
```

Comment:
not really related to stax porting, but `(bagl_element_t *)` cast not needed anymore.

Reply:
Done